Revamp the PPA reference guide to Diataxis standards (#511)
canonical/launchpad-manual: docs/user/how-to/packaging/request-ppa-architectures.rst

.. meta::
   :description: Enable additional build architectures for a Launchpad
      Personal Package Archive.

.. _request-ppa-architectures:

> Enable additional build architectures for a PPA
> ===============================================
> 
> Steps
> -----

Reviewer: ```suggestion
Enable additional build architectures for a PPA
===============================================
```
Two titles shouldn't follow each other without text between. For this doc, it may not be necessary to have additional subtitles due to its length. This is also a good candidate for merging with a related doc

1. From the PPA's page, open "Change details".
2. Tick the desired architectures. The full list of available
   architectures is at :ref:`ppa-architectures`.
3. Save.

> New uploads are then built for the enabled architectures.

Reviewer: ```suggestion
New uploads will be built for the enabled architectures.
```

Building existing sources for new architectures
-----------------------------------------------

Enabling an architecture affects only new uploads; sources already

> published in the PPA are not built for it. To build an

Reviewer: sources or binaries?

already-published source for a newly-enabled architecture, copy the

> source within the PPA: use the :ref:`copying-packages` workflow with

Reviewer: ```suggestion
source within the PPA. Use the :ref:`copying-packages` workflow with
```

"This PPA" as the destination, "The same series", and "Copy existing

> binaries". Launchpad then creates the missing builds, with no re-upload

Reviewer: ```suggestion
"This PPA" as the destination, and "The same series", and "Copy existing
binaries" selected. Launchpad will create the missing builds, with no re-upload
```
